src/query/plan/pretty_print.cpp

```diff
@@ -170,6 +170,11 @@ bool PlanPrinter::PreVisit(query::plan::RollUpApply &op) {
   return false;
 }
 
+bool PlanPrinter::PreVisit(query::plan::PeriodicCommit &op) {
+  WithPrintLn([&op](auto &out) { out << "* " << op.ToString(); });
+  return true;
```

Comment:
Should we add `op.input_->Accept(*this);`?

Reply:
PeriodicCommit is one branch only, PeriodicSubquery has 2 branches and I added it there.